khmer/_khmer.cc

```diff
@@ -5215,6 +5215,499 @@ static PyObject * hllcounter_merge(khmer_KHLLCounter_Object * me,
     Py_RETURN_NONE;
 }
 
+/********************************
+ * Assembler classes
+ ********************************/
+
+
+typedef struct {
+    PyObject_HEAD
+    LinearAssembler * assembler;
+} khmer_KLinearAssembler_Object;
+
+#define is_linearassembler_obj(v)  (Py_TYPE(v) == &khmer_KLinearAssembler_Type)
```

Comment:
Copypasta cruft ;) But, it's to check if a `PyObject *` you get from Python-land is of type `khmer_KLinearAssembler_Type`.